src/screens/Wallet/CreateWallet/CreateWalletStep0.tsx

```diff
@@ -26,99 +28,110 @@ export const CreateWalletStep0 = () => {
             >
-              recovery phrase
+              {t('backup.wallet.text')}{' '}
+              <Text
+                align="center"
+                onPress={showRecoveryModal}
+                fontSize={16}
+                fontFamily="Inter_500Medium"
+                color={COLORS.blue600}
+                style={{ textDecorationLine: 'underline' }}
+              >
+                {t('backup.wallet.popup')}
+              </Text>
+              <Text
+                align="center"
+                fontSize={15}
+                fontFamily="Inter_500Medium"
+                color={COLORS.nero}
+              >
+                {' '}
+                {t('make.sure.to.write.down')}
+              </Text>
             </Text>
-          </Button>
-        </View>
-        <Spacer value={verticalScale(12)} />
-        <Text
-          align="center"
-          fontSize={15}
-          fontFamily="Inter_500Medium"
-          color={COLORS.nero}
-        >
-          Make sure you have a pen and paper ready so you can write it down.
-        </Text>
-        <Spacer value={verticalScale(64)} />
-        <View style={{ alignSelf: 'center' }}>
-          <ElipseIcon />
+          </View>
+          <Spacer value={verticalScale(64)} />
+          <View style={styles.mnemoicLogo}>
+            <MnemoicIcon />
+          </View>
         </View>
-      </View>
-      <View style={{ marginBottom: verticalScale(100) }}>
-        <Row>
-          <CheckBox
-            fillColor={COLORS.sapphireBlue}
-            color={COLORS.white}
-            type="square"
-            onValueChange={setSelected}
-            value={selected}
-          />
-          <Spacer horizontal value={scale(12)} />
-          <Text fontSize={15} fontFamily="Inter_500Medium" color={COLORS.nero}>
-            I understand that if I lose my recovery phrase, AirDAO cannot
-            restore it.
-          </Text>
-        </Row>
-        <Spacer value={verticalScale(32)} />
-        <Button
-          disabled={!selected}
-          onPress={onContinuePress}
-          type="circular"
+        <View
           style={{
-            backgroundColor: selected ? COLORS.mainBlue : COLORS.neutralGray
+            marginBottom: verticalScale(120),
+            width: '100%'
           }}
         >
-          <Text
-            fontSize={16}
-            fontFamily="Inter_600SemiBold"
-            color={selected ? COLORS.white : COLORS.neutral600}
-            style={{ marginVertical: scale(12) }}
+          <Row>
+            <CheckBox
+              fillColor={COLORS.sapphireBlue}
+              color={COLORS.white}
+              type="square"
+              onValueChange={setSelected}
+              value={selected}
+            />
+            <Spacer horizontal value={scale(12)} />
+            <Text
+              fontSize={16}
+              fontFamily="Inter_500Medium"
+              color={COLORS.nero}
+              style={{ maxWidth: scale(330) }}
```

Comment:
@JavidHaji-zada wdyt if we leave it like that?

Reply:
I tried removing `maxWidth`, nothing changed. So, I think we can remove that

Reply:
If i remove it, on iphone 14 pro, this text doesn't split to next line and it's weird